example/src/Makefile

```diff
@@ -1,2 +1,15 @@
+ifeq ($(OS),Windows_NT)
+   TARGET = example.dll
+else
+	UNAME_S := $(shell uname -s)
+	ifeq ($(UNAME_S),Linux)
+		TARGET = example.so
+	endif
+	ifeq ($(UNAME_S),Darwin)
+		TARGET = example.dynlib
```

Comment:
Should this be the same as the `example.framework` as defined in the extension file? or should the extension file be `example.dynlib`, I'm not so familiar with macos shared libraries.

Reply:
https://stackoverflow.com/questions/2339679/what-are-the-differences-between-so-and-dylib-on-macos
Suggests this could even be a `.bundle`.

Reply:
https://github.com/godotengine/godot-cpp/blob/master/test/demo/example.gdextension
i found the extension from the godot-cpp binding ,their use .framework as they macos extension

Reply:
and this commit fix that https://github.com/readykit/gd/pull/3/commits/ba89eb9090dfa28863be833982795a5c421f4354